components/DEPS

```diff
@@ -10,3 +10,82 @@ include_rules = [
   # these dependencies; see /components/README.
   "-ipc",
 ]
+
+specific_include_rules = {
+  # IMPORTANT: Do not add new browser tests that depend on //chrome or
+  # //brave/browser/ to //brave/components.
+  #
+  # This is an exclusion list for browser tests written before proper dependency
+  # checking was implemented. These tests should be migrated out of
+  # //brave/components into //brave/browser eventually.
+  "android_page_appearing_browsertest\.cc": [
+    "+brave/browser",
```

Comment:
I missed that these should actually be `!`, not `+`. @zenparsing already has a PR removing the rewards tests, but we should update the others

Reply:
or since there are only a few left we could just move them